package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/mister-dave/playwright-demo#readme",
   "devDependencies": {
+    "@types/node": "^20.9.0",
```

Comment:
Added this in here since Typescript was not a fan of me doing `console.log(process.env.PW_USERNAME)`. Adding this small dev dependency makes it so typechecking works with those kinds of calls as well. You can remove this if you want and it may not break anything, but if you get some type errors when you try to compile this, then that may be a cause.